.deployment.config.json

```diff
@@ -1,6 +1,7 @@
 {
   "product": "ui-kit",
   "team_name": "searchui",
+  "development_workflow": "gitflow",
```

Comment:
Internal doc:
* https://tools-docs.dep.cloud.coveo.com/deployment-package/_static/schema_doc.html#development_workflow
* https://coveord.atlassian.net/wiki/spaces/CM/pages/1807712653/Following+up+with+a+deployment#For-gitflow-applications

Reply:
my bad, I meant that the `dev` pipeline is run on a alpha bump commit in the first place, I'll correct this. Alpha bumps are done prior to starting the pipeline as usual.